src/darwin/Framework/CHIP/MTRBaseDevice.h

```diff
@@ -605,22 +648,62 @@ typedef NS_ENUM(NSUInteger, MTREventPriority) {
 
 @interface MTREventReport : NSObject
 @property (nonatomic, readonly, copy) MTREventPath * path;
+
+/**
+ * eventNumber will only have a useful value if "error" is nil.
+ */
```

Comment:
My memory is rusty here, but are these doc comments of the "correct" form that will show up automatically in Xcode when you hover over the thing?

Reply:
I don't know.  They're the standard Matter comment style; if we need to change that we can, but we would probably need to change it across the board....